people/Eliška.json

```diff
@@ -0,0 +1,11 @@
+{
+  "key": "eliska",
+  "name": "Eliška Wichová",
+  "nick": "Eliska",
+  "email": "eliska.wichova@mild.blue",
+  "title_cs": "OPERATIONS MANAGER",
+  "title_en": "OPERATIONS MANAGER",
+  "short_desc_en": "Eliška holds a Master's degree in Landscape Engineering. She has various experience from corporate sector  IT and banking. She is the person who ensures	the smooth running of the MildBlue´s back office and business processes.",
+  "short_desc_de": "Eliška hat einen Abschluss als Ingenieur im Landschaftsbau. Sie verfügt über verschiedene Berufserfahrung Körperschaften in Umfeldern wie IT und Bankwesen. In Mild Blue sie ist verantwortlich für die reibungsloser Betrieb und Innovation im Bereich Backoffice un Geschäfte-Prozesse.",
```

Comment:
Eliška hat einen Abschluss als Ingenieur im Landschaftsbau. Sie verfügt über verschiedene Berufserfahrungen von Körperschaften in Umfeldern wie IT oder Bankwesen. In Mild Blue ist sie für einen reibungslosen Betrieb und Innovation im Bereich Backoffice und Geschäftsprozesse verantwortlich.